### ISSUE TITLE
Add Blazor UI generation support

### ISSUE THREAD

**Issue by a-urel** [EasyAbp/AbpHelper.CLI#119]
ABP Framework 3.2+ supports creating Blazor applications.
Blazor UI (razor components) generation is needed.

**Comment by wakuflair** [comment]
Yes Blazor will be supported, definitely!

**Comment by zhjmao** [comment]
waiting for it。

**Comment by wakuflair** [comment]
Most of the generating work are done.
But I'm waiting for "Dynamic Form for Blazor". (https://github.com/abpframework/abp/issues/5558)

**Comment by wakuflair** [comment]
Most of the generating work are done.
But I'm waiting for "Dynamic Form for Blazor". (https://github.com/abpframework/abp/issues/5558)

**Comment by omer-repo** [comment]
Thanks very much for this project. What about news on Blazor UI generation?

**Comment by wakuflair** [comment]
@omer-repo I agree with you that ABP team has no plan for "Dynamic Form for Blazor".
So I think it's better to generate Blazor form elements hard coded.
Maybe we could have dynamic form generating in future.

Any other suggestions?

**Comment by OaimIsGnay** [comment]
> abpframework/abp#5558

Thanks very much for this project.
How long will the blazer ui support.

**Comment by gdlcf88** [comment]
> How long will the blazer ui support.

We have heard your sound, but no plan yet. PRs are welcome.
